var/spack/repos/builtin/packages/py-shared-memory38/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PySharedMemory38(PythonPackage):
+    """Backport of multiprocessing.shared_memory for Python 3.6 and
+    3.7. Simply import all things from shared_memory to make your code
+    work. Note that multiprocessing.managers.SharedMemoryManager is also
+    included under shared_memory package."""
+
+    homepage = "https://github.com/mars-project/shared_memory38"
+    url = "https://pypi.io/packages/source/s/shared_memory38/shared_memory38-0.1.2.tar.gz"
```

Comment:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#id3

Reply:
oops, my template script is broken. fixed.